nsis/hs_err_pid3343656.log

```diff
@@ -0,0 +1,21 @@
+#
```

Comment:
This log must have been accidentally included.

Reply:
oops yes it was accidental. good catch